components/bl-data-grid/component.json

```diff
@@ -271,6 +271,30 @@
       "label": "Get Rows Data of ",
       "hasReturn": true,
       "color": "#204994"
+    },
+    {
+      "id": "sortByColIdAsc",
+      "label": "Sort By Column Id Asc in",
+      "inputs": [
+        {
+          "id": "columnId",
+          "label": "Column Id"
+        }
+      ]
+    },
+    {
+      "id": "sortByColIdDesc",
+      "label": "Sort By Column Id Desc in",
+      "inputs": [
+        {
+          "id": "columnId",
+          "label": "Column Id"
+        }
```

Comment:
should be `sortBy({columnId, direction})`

we doesn't need both actions

Also, could we add starting sorting? Or we already have it?

Reply:
Regarding default column sorting, I found a simpler way to do it

<img width="1013" alt="Screenshot 2023-08-25 at 00 40 37" src="https://github.com/Backendless/ui-builder-library/assets/56361878/eb51025e-968f-4e4a-9668-8fd3fdad6ab5">

In this way, the customer can choose the desired column, as well as the sorting direction.

I propose this variant because I could not implement as we discussed at the call. There were problems with the ref.

So, if we choose the variant I'm suggesting, the customers need to know somehow that there is an option to specify a default column sorting.